out/notebooks/commands.ts

```diff
@@ -112,6 +123,64 @@ export async function addVisualizationPreference(
     await controller.runCell(cell);
 }
 
+export async function enterEarliestTime(
```

Comment:
nit: not a big deal at all, but these two functions seem to share logic. Totally ok to keep them this way if refactoring is pain.

Reply:
I actually had the same thought and started to refactor it, but it ultimately only saved ~4 lines of code due to the multiple places where the two differ and ended up having a lot of indirection which made it harder to read so I reverted to this.

Reply:
Honestly, then this is way better :) 4 lines ain't worth it